app/src/main/java/com/futurice/freesound/feature/search/SearchViewModel.java

```diff
@@ -121,4 +125,10 @@ private static boolean isCloseEnabled(@NonNull final String query) {
         return TextUtils.isNotNullOrEmpty(query);
     }
 
+    @NonNull
+    public Observable<Option<Throwable>> getSearchErrorOnceAndStream() {
+        return lastErrorOnceAndStream
+                .hide()
```

Comment:
`hide` is used to prevent from casting Observable back to a subject. The thing is that `observeOn` already creates new observable and hides the subject. 
Remove `hide`